mobile-app/app/screens/AppNavigator/screens/Loans/screens/ConfirmCreateVaultScreen.tsx

```diff
@@ -105,6 +138,7 @@ function SummaryHeader (): JSX.Element {
           {translate('screens/ConfirmCreateVaultScreen', 'ID will generate once vault has been created')}
         </ThemedText>
       </View>
+      {props.conversion?.isConversionRequired === true && <ConversionTag />}
     </ThemedView>
   )
 }
```

Comment:
Commenting for line 157, for other confirm screen, when a conversion is required we display the `Transaction type` value as `Convert & create. vault`

Reply:
oh yeah haha ok will update